simulator/src/main/resources/app.properties

```diff
@@ -1 +1,3 @@
-
+#blockStream.delayBetweenBlockItems=2000000
```

Comment:
Should we standardize on milliseconds for this property? It's difficult to think in nanos.

Reply:
I can be explicit and say is NS on the property name itself.  And we can have a comment on rate `1_000_000 NS to 1 MS` conversion on the simulator docs (when we do them), however it has to be in `NS`, since in order to fine tune the rate of streaming to the BN and achieve 40K block items per second we need to have fine grain wait times that are sometimes in the hundreds of nanos.

Reply:
Ok, that makes sense.